docs/config/README.md

```diff
@@ -1078,6 +1078,37 @@ By default the module will be shown if any of the following conditions are met:
 format = "via [e $version](bold red) "
 ```
 
+## Fill
+
+The `fill` module fills any extra space on the line with a symbol. If multiple `fill` modules are
+present in a line they will split the space evenly between them. This is useful for aligning
+other modules.
+
+### Options
+
+| Option     | Default        | Description                            |
+| ---------- | -------------- | -------------------------------------- |
+| `symbol`   | `"."`          | The symbol used to fill the line.      |
```

Comment:
I went with a visible default because it makes it easier to understand what's going on when you first add a fill, but I agree that most people are going to want to use a space in the end, so happy to change it.